README.md

```diff
@@ -65,6 +65,20 @@ arrayToSentence(['Earth', 'Wind', 'Fire'], {
 }); //=> 'Earth, Wind & Fire'
 ```
 
+### options.oxfordComma
```

Comment:
The option name is confusing when you use this library for a non-English language; `separator` could not be a comma in that case.